common/CMakeLists.txt

```diff
@@ -30,7 +30,11 @@ target_link_libraries(common_path
 check_cxx_source_compiles("#include <filesystem>
   int main() { std::filesystem::path p(\"/\"); }" STD_FILESYSTEM_AVAILABLE)
 if(NOT "${STD_FILESYSTEM_AVAILABLE}" STREQUAL 1)
-  target_link_libraries(common_path PUBLIC stdc++fs)
+  if ("${CMAKE_SYSTEM_NAME}" STREQUAL "QNX")
```

Comment:
Q: Why can't this be in the `qnx.cmake` file?

Reply:
It can. Check out the last commit.